middleware/grpc_instrumentation.go

```diff
@@ -173,18 +171,115 @@ func (s *instrumentedClientStream) CloseSend() error {
 	return err
 }
 
-// errorCode converts an error into an error code string.
-func errorCode(err error) string {
-	if err == nil {
-		return "2xx"
+type InstrumentationLabelOption func(*InstrumentationLabel)
+
+var (
+	AcceptGRPCStatusesOption InstrumentationLabelOption = func(instrumentationLabel *InstrumentationLabel) {
```

Comment:
`AcceptGRPCStatusesOption` -> `ReportGRPCStatusOption`.

Also please add short documentation to it, since it's a public var.